test/fail_compilation/fix19246.d

```diff
@@ -0,0 +1,18 @@
+/* REQUIRED_ARGS: -de
+ * PERMUTE_ARGS:
+ * TEST_OUTPUT:
+---
+fail_compilation/fix19246.d(15): Deprecation: `0b_` isn't a valid integer literal, use `0b0` instead
+fail_compilation/fix19246.d(16): Deprecation: `0B_` isn't a valid integer literal, use `0B0` instead
+fail_compilation/fix19246.d(17): Deprecation: `0b` isn't a valid integer literal, use `0b0` instead
+---
+ */
+
+// https://issues.dlang.org/show_bug.cgi?id=19246
+
+void foo()
+{
+    auto a = 0b_;
+    auto b = 0B_;
+    auto c = 0b;
```

Comment:
Since both `0b_` and `0B_` are tested, should we test `0B` as well?

Reply:
Ping @wolframw we this fixed, this PR should be good to go.

Reply:
Added.